src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectSnapshotManager.cs

```diff
@@ -17,6 +17,8 @@ internal abstract class ProjectSnapshotManager : ILanguageService
 
         public abstract ProjectSnapshot GetLoadedProject(string filePath);
```

Comment:
Should remove this overload if the one below can be used everywhere

Reply:
Can't 😢 . There are cases when we just have the Roslyn project which then relies on us using the file path to lookup a snapshot